evopy/individual.py

```diff
@@ -38,3 +45,47 @@ def evaluate(self, fitness_function):
         self.fitness = fitness_function(self.genotype)
 
         return self.fitness
+
+    def _reproduce_single_variance(self):
+        """Create a single offspring individual from the set genotype and strategy parameters.
+
+        This function uses the single variance strategy.
+
+        :return: an individual which is the offspring of the current instance
+        """
+        new_genotype = self.genotype + \
+                       self.strategy_parameters[0] * np.random.randn(self.length)
+        scale_factor = np.random.randn() / (2 * self.length)
+        new_strategy = [max(self.strategy_parameters[0] * np.exp(scale_factor), self._EPSILON)]
```

Comment:
We should probably call this `new_strategy_parameters` or something similar, to prevent confusion with the `self.strategy`